frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -924,14 +979,34 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
           <>
             <Divider />
             <Row justify="center" align="middle">
-              <Button onClick={this.showAddVolumeLayerModal}>
+              <Button
+                onClick={this.showAddVolumeLayerModal}
+                style={{
+                  width: 235,
+                }}
+              >
                 <PlusOutlined />
                 Add Volume Annotation Layer
               </Button>
             </Row>
           </>
         ) : null}
 
+        {this.props.tracing.restrictions.allowUpdate && canBeMadeHybrid ? (
+          <Row justify="center" align="middle">
+            <Button
+              onClick={this.handleConvertToHybrid}
```

Comment:
Since this button now only has the "add skeleton layer" behavior, I'd rename `handleConvertToHybrid` to `addSkeletonAnnotationLayer`. Within that function, `convertToHybridTracing` can stay as is.